src/xenia/kernel/xboxkrnl/xboxkrnl_xconfig.cc

```diff
@@ -66,6 +70,68 @@ X_STATUS xeExGetXConfigSetting(uint16_t category, uint16_t setting,
     case 0x0003:
       // XCONFIG_USER_CATEGORY
       switch (setting) {
+#if XE_PLATFORM_WIN32
+          // TODO(gibbed): *REQUIRED BEFORE MERGING* all of the time zone stuff
+          // needs to get appropriate value from user settings / system.
```

Comment:
You could always just set the time zone to UTC by default. I've checked the other semi-modern emulators, and they barely even bother with time zone. RPCS3 defaults to UTC as well.